scripts/validate.py

```diff
@@ -49,6 +51,29 @@ def validate(src_path, schema_path):
         for item in items:
             try:
                 jsonschema.validate(instance=item, schema=schema)
+                # If it's an experiment we want to evaluate the branches
+                if "arguments" in item:
+                    validated = 0
+                    for branch in item.get("arguments").get("branches"):
+                        if "id" not in branch.get("value"):
+                            print("\tSkip branch", branch.get("slug"))
+                            validated = validated + 1
+                            continue
+                        # Try all of the available message schemas
+                        for try_schema_path in possible_schemas:
```

Comment:
Let's move this schema loading out of the loop, otherwise, it will repeatedly load/parse the file for every branch. Or you can cache the result for the following uses.